docs/serving/autoscaling/autoscale-go/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Autoscale Sample App - Go"
+linkTitle: "Autoscale Sample App - Go"
+weight: 100
+type: "docs"
+aliases:
+    - ./docs/serving/samples/autoscale-go/README.md
```

Comment:
Again, I think you want the URL on the site:

`/docs/serving/samples/autoscale-go` or `../../samples/autoscale-go`